tests/unit/LedgerLivePlatformSDK/index.spec.ts

```diff
@@ -368,17 +368,18 @@ describe("LedgerLivePlatformSDK/index.ts", () => {
         SDK.connect();
 
         const accountId = "accountId";
-        const message = "Test message";
+        const message = Buffer.from("Test message");
 
         const signedMessage = "Message signed";
 
         const spy = createSpyOnPostMessage(signedMessage);
 
         const res = await SDK.signMessage(accountId, message);
 
+        const expectedMessage = message.toString("hex");
         expect(res).to.deep.eq(signedMessage);
         expect(spy).to.have.been.called.with(
-          `{"jsonrpc":"2.0","method":"message.sign","params":{"accountId":"${accountId}","message":"${message}"},"id":1}`
+          `{"jsonrpc":"2.0","method":"message.sign","params":{"accountId":"${accountId}","message":"${expectedMessage}"},"id":1}`
```

Comment:
By using jest we could use `matchInlineSnapshot`, it would allow us to avoid testing implementation details like the `toString("hex")`

Reply:
Didn't know this one. I will update the test.